doc/source/examples/extended_examples/executable/executable.rst

```diff
@@ -0,0 +1,192 @@
+.. _executable_example:
+
+
+=======================================
+Create your own Python command line app
+=======================================
+
+This example shows how to create your own command line interface
+Python app which uses PyMAPDL to perform some simulations.
+This usage is quite convenient when aiming to automate workflows.
+One could build different PyMAPDL apps which can be called
+from the command-line tool with different arguments.
```

Comment:
```suggestion
This example shows how to create your own command-line app
in Python that uses PyMAPDL to perform some simulations.
This usage is quite convenient when automating workflows.
You can build different PyMAPDL apps that can be called
from the command line with different arguments.
```

Reply:
In line 5, you added an hyphen to the word ``command-line``, and here you removed. Why?